portfolio.html

```diff
@@ -52,31 +52,57 @@
 							
 				</section>
 		
-				<section class="projects"><div class="contentHeadings"><i class="fas fa-rocket fa-1.2x" style="color:#B4535A"></i> Projects</div>
+				<section class="projects"><div class="contentHeadings"><i class="fas fa-rocket fa-1.2x" style="color:#CE3B45"></i> Projects</div>
 				
-						<div class="experienceItemHeading"> Portfolio Project </div>
-						<div class="experienceItemBody"> I did a week's studing and Mayden accademy and all I got was this dumb profile page</div>
 				
+						<div class="portfolioBanner">
+							<div>
+								<div class="experienceItemHeading"> Portfolio Project </div>
+						
+								<div class="experienceItemBody"> I did a week's studing and Mayden accademy and all I got was this dumb profile page</div>
+							</div>
+							<div>	
+<!--This div is the portfolio image for display in regular mode - it disapears in mobile mode -->
+								<div class="folioImageHolder">
+									<a href="#" >
+										<img class= "folioImage"src="images/PortfolioScreenshot.jpg" alt="a screenshot of this home page">
+									</a>
+									<figcaption>This page, right here.</figcaption>
+								</div>
+<!--this following div is an icon link that replaces the portfolio image in mobile mode -->
+								<div class="folioMobileLink">
+									<a href="#" class="mobileLink"><i class="fas fa-arrow-circle-right fa-2x"></i></a>
+								</div>
+							</div>
+						</div>
 				</section>
 				
 <!--right hand side of the page content -->
 		</div>
 		<div class="contactBar">
 			<div class="header">
-				<div><img src="images/harryicon.jpeg" class="profilePic" alt="Harry smiling handsomely"></div>
+				<div><img src="images/harryicon.jpeg" class="profilePic" alt="Harry framed and illuminated like a religious icon"></div>
```

Comment:
start new html tag on its own line, also dont forget to add closing slash to the self closing tag eg: `<img />`